packages/web-react/src/components/FileUploader/FileUploaderInput.tsx

```diff
@@ -67,9 +64,12 @@ const FileUploaderInput = (props: SpiritFileUploaderInputProps) => {
     validationState,
   });
   const { styleProps, props: transferProps } = useStyleProps(restProps);
-
   const [ids, register] = useAriaIds(ariaDescribedBy);
 
+  useEffect(() => {
```

Comment:
Are we able to test this?

Reply:
https://stackoverflow.com/questions/75376392/how-to-test-server-side-render-with-testing-library-react

It would be nice to have 2 tests. One for CSR where event listeners like `onDragOver`, etc. are set, and one for SSR where there are undefined. :-)

Reply:
I'm sorry, but I still can't figure out how to test it :(

Reply:
solved